src/libraries/System.Net.Http.WinHttpHandler/tests/FunctionalTests/WinHttpHandlerTest.cs

```diff
@@ -97,19 +96,36 @@ public async Task SendAsync_SlowServerAndCancel_ThrowsTaskCanceledException()
             }
         }
 
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/20675")]
         [OuterLoop]
         [Fact]
-        [OuterLoop]
-        public void SendAsync_SlowServerRespondsAfterDefaultReceiveTimeout_ThrowsHttpRequestException()
+        public async void SendAsync_SlowServerRespondsAfterDefaultReceiveTimeout_ThrowsHttpRequestException()
         {
             var handler = new WinHttpHandler();
             using (var client = new HttpClient(handler))
             {
-                Task<HttpResponseMessage> t = client.GetAsync(SlowServer);
+                var triggerResponseWrite = new TaskCompletionSource<bool>(TaskCreationOptions.RunContinuationsAsynchronously);
```

Comment:
We have non-generic TCS, you don't need the boolean. Also any reason why `RunContinuationsAsynchronously`?

Reply:
I wasn't aware of the non-generic TCS, will change it